winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+
+#### Enabling Filters
+
+You can show filters in exported worksheet by enabling filter for the exported range in the worksheet.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+workBook.Worksheets[0].AutoFilters.FilterRange = workBook.Worksheets[0].UsedRange;
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-filter-on-exported-excel.png" alt="Filters on Exported Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+While using `stacked headers`, the filter option enable for stacked header cell. To avoid this, you can specify the `range` based on Stacked headers count to show the filter icon in the header cell.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CanExportStackedHeaders = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+var range = "A" + (dataGrid.StackedHeaderRows.Count + 1).ToString() + ":" + workBook.Worksheets[0].UsedRange.End.AddressLocal;
+excelEngine.Excel.Workbooks[0].Worksheets[0].AutoFilters.FilterRange = workBook.Worksheets[0].Range[range];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+You can refer [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/worksheet-cells-manipulation#data-filtering).
+
+#### Customize the range of cells
+
+You can customize the range of cells after exporting to excel by directly manipulating worksheet.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+workBook.Worksheets[0].Range["A2:A6"].CellStyle.Color = Syncfusion.Drawing.Color.LightSlateGray;
+workBook.Worksheets[0].Range["A2:A6"].CellStyle.Font.Color = ExcelKnownColors.White;
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-range-of-cells-customization.png" alt="Customizing Range of Cells in Exported Excel for WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Performance
+
+Using `DataGridExcelExportOptions.CellsExportHandler` and changing settings for each cell will consume more memory and time consumption. So, avoid using `CellsExportHandler` and instead of you can do the required settings in the exported sheet.
+ 
+### Formatting column without using CellsExportHandler
+
+You can perform cell level customization such as row-level styling, formatting particular column in the exported worksheet. 
```

Comment:
Perform cell-level customization such as row-level styling and formatting particular columns in the exported worksheet.

Reply:
Content modified